### PR TITLE
Rearrange the docs structure

### PR DESCRIPTION
**Pull request by stefanprodan**

### CODE REVIEW

File: content/en/blog/2021-04-29-may-update.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The Flux v2 CLI and the GitOps Toolkit controllers are now CII Best
 Practices certified.
 
 Checkout the [new bootstrap
-procedure](/docs/guides/installation/#generic-git-server).
+procedure](/docs/installation/_index.md#generic-git-server).
```
Comment (dholbach) [inline content/en/blog/2021-04-29-may-update.md:66]: General question: do you envision more docs under `/installation/`?

File: content/en/docs/migration/flux-v1-migration.md [hunk 2]
```diff
@@ -66,7 +66,7 @@ repository. The Git repository created during bootstrap can be used
 to define the state of your fleet of Kubernetes clusters.
 
 For a detailed walk-through of the bootstrap procedure please see the [installation
-guide]({{< relref "../guides/installation.md" >}}).
+guide](ç).
```
Comment (dholbach) [inline content/en/docs/migration/flux-v1-migration.md:69]: This looks like a typo?

**Review by dholbach — CHANGES_REQUESTED** [review]
Looks good apart from the typo.

File: content/en/blog/2021-04-29-may-update.md (hunk 1 above)
Comment (stefanprodan) [reply content/en/blog/2021-04-29-may-update.md:66]: Yes, that page is already too big and we'll need to add more stuff to it.

Comment (dholbach) [reply content/en/blog/2021-04-29-may-update.md:66]: Super!

File: _redirects [hunk 3]
```diff
@@ -3,3 +3,4 @@
 /docs/components/kustomize/controller       /docs/components/kustomize      301!
 /docs/components/notification/controller    /docs/components/notification   301!
 /docs/components/source/controller          /docs/components/source         301!
+/docs/guides/installation                   /docs/installation              301!
```
Comment (dholbach) [inline _redirects:6]: I'm quite certain this already worked in the past, but now it doesn't.

Apparently the problem is that we don't deploy `_redirects` into `/` (cf https://docs.netlify.com/routing/redirects/#syntax-for-the-redirects-file)

The easiest way to do this now is:

```cli
git mv _redirects static/
```

If you move it as part of this PR, I'll make sure to document it as part of #237.

Comment (stefanprodan) [reply _redirects:6]: Done

Comment (dholbach) [reply _redirects:6]: Thanks a lot - it works now! https://609bc5434a75050007e1eb17--fluxcd.netlify.app/docs/guides/installation

**Review by dholbach — APPROVED** [review]
Nice work!

File: _redirects (hunk 3 above)
Comment (dholbach) [reply _redirects:6]: I put it into #256

**Pull request merged**